Systancia/cleanroom/ingest/parser.yml

```diff
@@ -0,0 +1,65 @@
+name: systancia-cleanroom
+pipeline:
+  - name: event
+    external:
+      name: grok.match
+      properties:
+        input_field: "original.message"
+        output_field: result
+        pattern: "%{FALLBACK}"
+        custom_patterns:
+          BASE_INFO: '(\s*(org:%{DATA:organization_name}),?)?(\s*(user:%{EMAILADDRESS:user_email}),?)(\s*(ip:%{IP:source_ip}))'
+          FALLBACK: "%{DATA:event_reason}%{BASE_INFO}"
+
+  - name: event_reason
+    filter: "{{event.result.get('event_reason') != None}}"
+    external:
+      name: grok.match
+      properties:
+        input_field: "{{event.result.event_reason}}"
+        output_field: result
+        pattern: "%{SESSION_SUCCESS}|%{SESSION_FAILURE}|%{PROCESS}"
+        custom_patterns:
+          PROCESS: "%{WORD:process_event} %{DATA} application%{GREEDYDATA}"
+          SESSION_FAILURE: "%{WORD:session_event} de la connexion au portail web%{GREEDYDATA}"
+          SESSION_SUCCESS: '%{WORD:session_event} (%{DATA})?(\s*(plugin utilisateur)|\s*(portail web))%{GREEDYDATA}'
+
+  - name: set_default_fields
+
+stages:
+  set_default_fields:
+    actions:
+      - set:
+          event.reason: "{{event.result.event_reason}}"
+          observer.vendor: "Systancia"
+          observer.product: "Systancia Cleanroom"
+          organization.name: "{{event.result.organization_name}}"
+          user.email: "{{event.result.user_email}}"
```

Comment:
My bad. Can we split `user.email` into `user.name` and `user.domain` (more suitable I think)?